layers/+lang/clojure/packages.el

```diff
@@ -115,8 +117,9 @@
             "sE" 'spacemacs/cider-send-last-sexp-to-repl-focus
             "sf" 'spacemacs/cider-send-function-to-repl
             "sF" 'spacemacs/cider-send-function-to-repl-focus
-            "si" 'cider-jack-in
-            "sI" 'cider-jack-in-clojurescript
+            "sjc" 'cider-jack-in-clj
+            "sjf" 'cider-jack-in-clj$cljs
```

Comment:
```suggestion
            "sjf" 'cider-jack-in-clj&cljs
```
I'm guessing that the `$` (dollar sign) is a typo.

Because both the changelog and readme key binding table says that the 
`SPC m s j f` key binding calls `cider-jack-in-clj&cjls` with an `&` (ampersand).